testing/js_test_report.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
I assume we'll need to keep this updated when we change the HTML structure of the report generated?